builder/dockerfile/internals.go

```diff
@@ -539,9 +539,7 @@ func (b *Builder) create() (*container.Container, error) {
 
 func (b *Builder) run(c *container.Container) error {
 	var errCh chan error
-	if b.Verbose {
-		errCh = c.Attach(nil, b.Stdout, b.Stderr)
-	}
+	errCh = c.Attach(nil, b.Stdout, b.Stderr)
```

Comment:
@ripcurld00d if you remove the `b.Verbose` condition here, why not remove it when receiving from `errCh` couple lines below?